calico/felix/test/test_config.py

```diff
@@ -208,25 +210,31 @@ def test_file_sections(self):
             self.assertEqual(config.IFACE_PREFIX, "whatever")
             self.assertEqual(config.METADATA_PORT, 246)
             self.assertEqual(config.METADATA_IP, "1.2.3.4")
+            self.assertEqual(config.REPORTING_INTERVAL_SECS, 5)
+            self.assertEqual(config.REPORTING_TTL_SECS, 12)
 
     def test_env_var_override(self):
         """
-        Test environment variables override config options,
+        Test environment variables override config options.
         """
         with mock.patch.dict("os.environ", {"FELIX_ETCDADDR": "9.9.9.9:1234",
-                                            "FELIX_METADATAPORT": "999"}):
+                                            "FELIX_METADATAPORT": "999",
+                                            "FELIX_REPORTINGINTERVALSECS": "10"}):
             with mock.patch('calico.common.complete_logging'):
                 config = Config("calico/felix/test/data/felix_section.cfg")
 
         host_dict = { "InterfacePrefix": "blah",
                       "StartupCleanupDelay": "42",
                       "MetadataAddr": "4.3.2.1",
-                      "MetadataPort": "123" }
+                      "MetadataPort": "123",
+                      "ReportingIntervalSecs": "10",
+                      "ReportingTTLSecs": "20"}
 
         global_dict = { "InterfacePrefix": "blah",
                         "StartupCleanupDelay": "99",
                         "MetadataAddr": "5.4.3.2",
-                        "MetadataPort": "123" }
+                        "MetadataPort": "123",
+                        "ReportingIntervalSecs": "10"}
```

Comment:
Again, this needs to be different to the environment interval value for the test to be useful.